### PR TITLE
Improvements to export_dashboards.py

### PR DESCRIPTION
**Pull request by monicasarbu**
This PR includes couple of improvements that are needed after the latest changes in the way we import the Kibana dashboards in Elasticsearch:

- [x] export only the Kibana dashboards, not the index pattern as it can overwrite the generated index pattern
- [x] pass a regex to the command line to match the dashboards that need to be exported
- [x] delete dev-tools/import_dashboards.sh and dev-tools/import_dashboards.ps
- [x] update docs

### CODE REVIEW

File: dev-tools/export_dashboards.py [hunk 1]
```diff
@@ -97,35 +91,26 @@ def main():
         description="Export the Kibana dashboards together with"
                     " all used visualizations, searches and index pattern")
     parser.add_argument("--url",
-                        help="Elasticsearch URL. E.g. http://localhost:9200",
+                        help="Elasticsearch URL. By default: http://localhost:9200",
                         default="http://localhost:9200")
-    parser.add_argument("--beat",
-                        help="Beat name e.g. metricbeat",
+    parser.add_argument("--regex",
```
Comment (ruflin) [inline dev-tools/export_dashboards.py]: Nice

File: libbeat/docs/newdashboards.asciidoc [hunk 2]
```diff
@@ -89,59 +89,70 @@ The Elasticsearch index pattern where Kibana saves its configuration. The defaul
 [[export-dashboards]]
 === Exporting New and Modified Beat Dashboards
 
-To export all the dashboards for a Beat, including any new or modified dashboards and all dependencies such as
-visualizations, searches, and index patterns, you can use the Python script `export_dashboards.py` from  
-https://github.com/elastic/beats/tree/master/dev-tools[dev-tools]. Before running the script for the first time, you
-need to create an environment that contains all the required Python packages. See the dev-tools
+To export all the dashboards for any Elastic Beat or any community Beat, including any new or modified dashboards and all dependencies such as
+visualizations, searches, and index patterns, you can use the Python script `export_dashboards.py` from
```
Comment (ruflin) [inline libbeat/docs/newdashboards.asciidoc]: remove index patterns here

File: libbeat/scripts/Makefile [hunk 3]
```diff
@@ -270,12 +270,12 @@ docs-preview:
 ### KIBANA FILES HANDLING ###
 ES_URL?=http://localhost:9200
 
-.PHONY: export-dashboards
-export-dashboards: python-env
-	. ${PYTHON_ENV}/bin/activate && python ${ES_BEATS}/dev-tools/export_dashboards.py --url ${ES_URL} --dir $(shell pwd)/etc/kibana --beat ${BEATNAME}
+.PHONY: export-dashboards 
+export-dashboards: python-env update
+	. ${PYTHON_ENV}/bin/activate && python ${ES_BEATS}/dev-tools/export_dashboards.py --url ${ES_URL} --dir $(shell pwd)/etc/kibana --regex ${BEATNAME}-*
```
Comment (ruflin) [inline libbeat/scripts/Makefile]: Isn't the correct regexp `${BEATNAME}-*` as you have in the example `metricbeat*`

File: libbeat/docs/newdashboards.asciidoc (hunk 2 above)
Comment (monicasarbu) [inline libbeat/docs/newdashboards.asciidoc]: thanks, I will update the PR with the docs fixes.

File: libbeat/scripts/Makefile [hunk 4]
```diff
@@ -270,12 +270,12 @@ docs-preview:
 ### KIBANA FILES HANDLING ###
 ES_URL?=http://localhost:9200
 
-.PHONY: export-dashboards
-export-dashboards: python-env
-	. ${PYTHON_ENV}/bin/activate && python ${ES_BEATS}/dev-tools/export_dashboards.py --url ${ES_URL} --dir $(shell pwd)/etc/kibana --beat ${BEATNAME}
+.PHONY: export-dashboards 
+export-dashboards: python-env update
```
Comment (ruflin) [inline libbeat/scripts/Makefile]: Why is the update needed before running this one?

**Pull request merged**